tests/waku_archive/test_driver_queue.nim

```diff
@@ -18,13 +18,8 @@ proc genIndexedWakuMessage(i: int8): (Index, WakuMessage) =
   let
     message = WakuMessage(payload: @[byte i], timestamp: Timestamp(i))
     topic = "test-pubsub-topic"
-    cursor = Index(
-      receiverTime: Timestamp(i),
-      senderTime: Timestamp(i),
-      digest: MessageDigest(data: data),
-      pubsubTopic: topic,
-      hash: computeMessageHash(topic, message),
-    )
+    cursor =
+      Index(time: Timestamp(i), hash: computeMessageHash(topic, message), topic: topic)
```

Comment:
I wonder if it'd make sense to keep using the `pubsubTopic` name to make it more explicit
```suggestion
      Index(time: Timestamp(i), hash: computeMessageHash(topic, message), pubsubTopic: topic)
```

Reply:
W8 is misread that. Yes i will take note of this and update it in #2761